x/cheqd/tests/constants.go

```diff
@@ -1,16 +1,18 @@
 package tests
 
 const (
-	AliceDID    = "did:cheqd:test:alice"
-	BobDID      = "did:cheqd:test:bob"
-	CharlieDID  = "did:cheqd:test:charlie"
-	AliceKey1   = AliceDID + "#key-1"
-	AliceKey2   = AliceDID + "#key-2"
-	BobKey1     = BobDID + "#key-1"
-	BobKey2     = BobDID + "#key-2"
-	BobKey3     = BobDID + "#key-3"
-	BobKey4     = BobDID + "#key-4"
-	CharlieKey1 = CharlieDID + "#key-1"
-	CharlieKey2 = CharlieDID + "#key-2"
-	CharlieKey3 = CharlieDID + "#key-3"
+	AliceDID      = "did:cheqd:test:00000000000alice"
+	BobDID        = "did:cheqd:test:0000000000000bob"
+	CharlieDID    = "did:cheqd:test:000000000charlie"
```

Comment:
Should this be testnet (to exercise namespace check)? Basically, using `test` as the namespace is fine, as long as that's the same namespace used in the test. For example, if namespace is set to "mainnet", a user shouldn't be able to create a DID with namespace "testnet" or "test" and so on. (I don't know whether we check this already, if not, the probably needs to be its own task next sprint.)